GDJS/Runtime/howler-sound-manager/howler-sound-manager.js

```diff
@@ -44,16 +48,21 @@ gdjs.HowlerSound = function (o) {
   // sync'ed with the sound - though this should be redundant
   // with `play`/`pause` methods already doing that. Keeping
   // that to be sure that the status is always correct.
-  this.on('play', function () {
+  this._howl.on('play', function () {
     that._paused = false;
     that._stopped = false;
   });
-  this.on('pause', function () {
+  this._howl.on('pause', function () {
     that._paused = true;
     that._stopped = false;
   });
 };
-gdjs.HowlerSound.prototype = Object.create(Howl.prototype);
+
+for(var item in Howl.prototype) {
+  gdjs.HowlerSound.prototype[item] = function() {
+    this._howl[item].apply(this, [arguments, this.id]);
+  }
+}
```

Comment:
While it works, I'm a bit concerned that JS engine could very well de-optimize the whole thing. 
(Basically, if you use `prototype`, it's usually for Object.create which is how you do inheritance in ES5 - or for declaring a method. Otherwise, I'm afraid of JS engine not recognizing what you do and disabling their optimizations)

Instead, let's write simple, boring code: just declare a bunch of methods that call into `this._howl`.
Not as "clever" but totally safe, works with autocompletion, works with static typing, and the reader of the code understands everything by reading once instead of wondering what is going on with this weird loop on `prototype` :)